README.md

```diff
@@ -1,69 +1,107 @@
-# Bonsai Foundry Template
+# RISC Zero Foundry Template
 
 > **Note: This software is not production ready. Do not use in production.**
 
-Starter template for writing an application using [Bonsai].
+Starter template for writing an application using [RISC Zero] and Ethereum.
 
-This repository implements an application on Ethereum utilizing Bonsai as a [coprocessor] to the smart contract application.
-It provides a starting point for building powerful new applications on Ethereum that offload computationally intensive, or difficult to implement, tasks to be proven by the [RISC Zero] [zkVM], with verifiable results sent to your Ethereum contract.
-
-*For a 60 second overview of how this template and off-chain computation with Bonsai work, [check out the video here](https://www.youtube.com/watch?v=WDS8X8H9mIk).*
+This repository implements an application on Ethereum utilizing RISC Zero as a [coprocessor] to the smart contract application.
+It provides a starting point for building powerful new applications on Ethereum that offload computationally intensive (i.e. gas expensive), or difficult to implement.
```

Comment:
Maybe this would read better?

```suggestion
It provides a starting point for building powerful new applications on Ethereum that can benefit from offloading computationally intensive tasks (i.e. gas expensive), or code logic that would be difficult to implement in Solidity.
```

Reply:
Thanks for finding this mangled sentence. I think the suggestion is a bit verbose. I wrote up a version here that I think fixes the issue while being a bit more direct in language. (I also decided "difficult to implement" could use an example).
```suggestion
It provides a starting point for building powerful new applications on Ethereum that offload computationally intensive (i.e. gas expensive), or would be difficult to implement Solidity (e.g. ed25519 signature verification, or HTML parsing).
```